libraries/AP_LandingGear/AP_LandingGear.h

```diff
@@ -39,7 +39,14 @@ class AP_LandingGear {
     void init();
 
     /// returns true if the landing gear is deployed
-    bool deployed() const { return _deployed; }
+    bool deployed();
+    
+    enum LG_WOW_State {
+        LG_UNKNOWN = -1,
+        LG_NO_WOW = 0,
+        LG_WOW = 1,
+        };
+    int8_t wow();
```

Comment:
This should return `LG_WOW_State`. I would also prefer the method had the full name, pretty sure wow means nothing to most people.